### ISSUE TITLE
Installing from 2.0-rc1 branch?

### ISSUE THREAD

**Issue by rnkn** [jrnl-org/jrnl#453]
I've been using a `jrnl` install via Homebrew, but I'd like to start using 2.0, if possible.

Can you offer some brief guidance on installing from the `2.0-rc1` branch?

**Comment by Rafi993** [comment]
I would like to install 2.0 since the temp file issue in windows is fixed with pull-request https://github.com/maebert/jrnl/pull/323. How do I proceed there seems to be no instruction to do so. the version 2.0-rc1 is also not in pip

**Comment by rnkn** [comment]
@Rafi993 I gave up and switched to using a basic txt file with gpg. I use Emacs so I have these file variables at the bottom:
~~~
;; Local Variables:
;; mode: outline
;; outline-regexp: "[0-9]\\{4\\}-[0-9]\\{2\\}-[0-9]\\{2\\}"
;; eval: (olivetti-mode t)
;; eval: (adaptive-wrap-prefix-mode t)
;; End:
~~~
Works just as well as jrnl, but I’ll reopen in case the developer ever responds.

**Comment by wren** [comment]
Hi, you can currently install v2.0-rc2 via pip with this command `pip install jrnl==2.0.0rc2`. You'll get all the fixes that come with v2.0. Also, please keep in mind that it's a release candidate, so please report any bugs you find. And make a backup, just in case!
